### PR TITLE
Script to upgrade a Centrifuge parachain live

### PR DESCRIPTION
**Pull request by gpmayorga**
# Description

Upgrade a tesnet Centrifuge runtime by connecting to the node and uploading a new wasm using JS

Tested with sudo rights on development and demo.

In case we have control over the council on Altair or we have a parachain testnet with council and democracy in the future I left the code that allows for performing a RU using council keys

### CODE REVIEW

File: scripts/js/runtime-upgrade-remote/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,853 @@
+{
```
Comment (lemunozm) [inline scripts/js/runtime-upgrade-remote/package-lock.json:1]: I was wondering if we needed to commit this, and yes, we need it: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Comment by gpmayorga** [comment]
We'll wait for @wischli to try this out and upgrade Demo, if it works I'd consider it ready

**Comment by gpmayorga** [comment]
Since we don't have an environment where we control the council keys, I'm going to remove the council motion from the automation but leave the govermnent proposal

File: scripts/js/runtime-upgrade-remote/package-lock.json (hunk 1 above)
Comment (wischli) [reply scripts/js/runtime-upgrade-remote/package-lock.json:1]: I would prefer the `yarn.lock` to follow the `runtime-upgrade-local` path. AFAIK, yarn locks are preferred.

Comment (wischli) [reply scripts/js/runtime-upgrade-remote/package-lock.json]: done in 4e38fda8b5550d955907c227ca518476171054bc

**Review by wischli — APPROVED** [review]
@gpmayorga Thanks a lot for making our lives easier!

I removed the council related code paths because in the foreseeable future, we don't need to upgrade a non-local chain without sudo. If we need it at some point, we can migrate it over from the local script.

Later this year, I will refactor both scripts to use the same code paths as mentioned in the sync to decrease maintenance overhead. In the end, both scripts are very very similar. For now, there are higher priorities and this script worked well for Dev and Demo.

**Pull request merged**